doc/api/cli.md

```diff
@@ -184,6 +184,22 @@ added: v6.0.0
 Force FIPS-compliant crypto on startup. (Cannot be disabled from script code.)
 (Same requirements as `--enable-fips`.)
 
+### `--frozen-intrinsics`
+<!-- YAML
+added: REPLACEME
+-->
+
+Enable experimental frozen intrinsics like `Array` and `Object`.
+
+Support is currently only provided for the root context and no guarantees are
+currently provided that `global.Array` is indeed the default intrinsic
+reference.
+
+Limited support for subclassing builtins is provided under this flag due to 
+ECMA262 bug https://github.com/tc39/ecma262/pull/1320.
+
+Both of the above may change in future updates, which will be breaking changes.
```

Comment:
I've more carefully documented the support here.

Reply:
This should likely include a note that breakage in the ecosystem *is* likely given how extensively userland has monkeypatched or polyfilled various intrinsics... `Promise` for instance...

Reply:
Great point, I've added the following note here:

> **Code breakage is highly likely with this flag**, especially since limited
support for subclassing builtins is provided currently due to ECMA-262 bug
https://github.com/tc39/ecma262/pull/1320.